gpu/install_gpu_drivers.sh

```diff
@@ -1,92 +1,136 @@
 #!/bin/bash
 
-install_gpu() {
-
-    # Set cuda nvidia repository
-    curl -O https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-repo-ubuntu1804_10.0.130-1_amd64.deb
-    sudo dpkg -i cuda-repo-ubuntu1804_10.0.130-1_amd64.deb
-    sudo apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/7fa2af80.pub
-
-    # install cuda
-    sudo apt update
-    sudo apt-get -y install cuda
-
-    echo "################################# Validate NVIDIA-SMI #####################################"
-    nvidia-smi
-    nvidia_version=$(nvidia-smi | grep NVIDIA-SMI | grep CUDA)
-
-    if [[ "${nvidia_version}" == *"CUDA Version"* && "${nvidia_version}" == *"Driver Version"* ]]; then
-
-        # install nvidia-docker based on distribution
-        distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
-        curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add -
-        curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.list | sudo tee /etc/apt/sources.list.d/nvidia-docker.list
-        sudo apt-get update && sudo apt-get install -y nvidia-docker2
-
-        # update docker daemon json with nvidia runtime
-        set_daemon_json
-        sudo systemctl restart docker
+function install_gpu() {
+    #Install CUDA Drivers for Ubuntu18
+    os=$(. /etc/os-release;echo $ID)
+    distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
+    major_version=$(. /etc/os-release;echo $VERSION_ID | cut -d '.' -f1)
+
+    if [[ "${os}" = "ubuntu" && "${major_version}" = "18" ]];
```

Comment:
We won't support ubuntu 16 as well?

Reply:
yeah we don't right now , we have to enhance the script to do so